src/main/webapp/resources/js/apis/metadata/metadata-import.js

```diff
@@ -42,11 +42,12 @@ export const metadataImportApi = createApi({
       invalidatesTags: ["MetadataImport"],
     }),
     saveProjectSampleMetadata: build.mutation({
-      query: ({ projectId, sampleNameColumn }) => ({
+      query: ({ projectId, sampleNames }) => ({
```

Comment:
All of these urls contain `/upload` so you should just probably add that to your base url.

Reply:
Please see ac4e505453ef442dca7f740b01d74dcf74010739.